src/osx/carbon/font.cpp

```diff
@@ -934,8 +934,10 @@ CGFloat wxNativeFontInfo::GetCTSlant(CTFontDescriptorRef descr)
     return slant;
 }
 
+// common prefix of plist serializiation, gets removed and readded
+static const wxString s_plistPrefix = "<?xml version=\"1.0\" encoding=\"UTF-8\"?><!DOCTYPE plist PUBLIC "
+    "\"-//Apple//DTD PLIST 1.0//EN\" \"http://www.apple.com/DTDs/PropertyList-1.0.dtd\">";
```

Comment:
I think creating global `wxString`s does work right now, but we've had problems with this in the past, so I think it would be still better to replace it with an accessor function using static variable inside it, to avoid constructing it before wx is initialized.

Reply:
ok, done, thanks